codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapperGenerator.java

```diff
@@ -116,6 +116,32 @@ public SolidityFunctionWrapperGenerator(
                 abiFuncs);
     }
 
+    public SolidityFunctionWrapperGenerator(
```

Comment:
Let's extend visibility of the existing constructor, instead of creating a new one.

Reply:
That's fine by me, even easier (I thought there was a reason to not make it public), you an go ahead.

Reply:
Great, let's make it and merge this PR.